bitnami/postgresql/templates/svc-read.yaml

```diff
@@ -23,6 +23,7 @@ spec:
   type: {{ $serviceType }}
   {{- if and $serviceLoadBalancerIP (eq $serviceType "LoadBalancer") }}
   loadBalancerIP: {{ $serviceLoadBalancerIP }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy | quote }}
```

Comment:
I would try to make the definition of the externalTrafficPolicy optional, which would help users using an old values.yaml, from a previous version that would be missing this new key.

```YAML
{{- if .Values.externalTrafficPolicy }}
externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy | quote }}
{{- end }}
```

Reply:
I've just copied the code from the mysql chart (#5498) to keep it simple.

You don't need to have it in your local values, as it will have a default value in this chart.
`Cluster`  is the default valkue automatically set by kubernetes if no `externalTrafficPolicy` is defined. So this is also not breaking anybody.

But if required, i can make this optional.

Reply:
Ok, thanks for checking this, solving the comments then and merging your request! 👍